docs/release_notes.rst

```diff
@@ -51,6 +51,14 @@ EIA 176
 * Extracted these interim tables up through the latest 2023 data release. See
   :issue:`4002` and :pr:`4004`.
 
+EIA 930
+~~~~~~~
+* Updated the EIA 930 data to include data published up through 2025half1. See
+  :issue:`4040` and PR :pr:`4054`. 10 new energy sources were added and 3 were
+  retired; see
+  :doc:`data_sources/eia930#changes-in-energy-source-granularity-over-time` for
+  more information.
```

Comment:
the data is published in the half year files but not all the data for jan through june is published yet.

```suggestion
* Updated the EIA 930 data to include data published up through the beginning of
  February 2025. See :issue:`4040` and PR :pr:`4054`. 10 new energy sources
  were added and 3 were retired; see
  :doc:`data_sources/eia930#changes-in-energy-source-granularity-over-time` for
  more information.
```